src/MaplibreLayerRenderer.ts

```diff
@@ -0,0 +1,170 @@
+import {FrameState} from 'ol/Map';
+import {toDegrees} from 'ol/math';
+import {toLonLat} from 'ol/proj';
+import LayerRenderer from 'ol/renderer/Layer';
+import GeoJSON from 'ol/format/GeoJSON';
+import {Coordinate} from 'ol/coordinate';
+import {FeatureCallback} from 'ol/renderer/vector';
+import {Feature} from 'ol';
+import {Geometry, SimpleGeometry} from 'ol/geom';
+import {Pixel} from 'ol/pixel';
+import MapLibreLayer from './ol-maplibre-layer';
+
+const VECTOR_TILE_FEATURE_PROPERTY = 'vectorTileFeature';
+
+/**
+ * @private
```

Comment:
This comment is not useful. And please, remove others not useful comments.
(useful comments and documentation is still welcome)